src/NodaTime/Instant.cs

```diff
@@ -533,11 +534,20 @@ public static Instant Max(Instant x, Instant y)
             BclEpoch.PlusTicks(dateTimeOffset.Ticks - dateTimeOffset.Offset.Ticks);
 
         /// <summary>
-        /// Converts a Julian Day Number into a new Instant representing the same instant in time
+        /// Converts a Julian Date (JD) into a new <see cref="Instant"/> representing the given number of days
```

Comment:
That's the abbreviation used by Wikipedia, compared with JDN which is the integer Julian Date Number. Happy to remove it if it doesn't add anything though.